js/settingsHelpers.js

```diff
@@ -597,7 +637,7 @@ const updateFirebaseAuthPhone = async (phone) => {
  * @param {String} providerType - 'email' or 'phone' 
  * @returns {boolean} - true if the unlink was successful, false otherwise
  */
-export const unlinkFirebaseAuthProvider = async (providerType, userData) => {
+export const unlinkFirebaseAuthProvider = async (providerType, userData, newPhone, isPhoneRemoval) => {
```

Comment:
Do we have to update Jsdocument above for new parameters?

Reply:
Good call! Done.